src/hooks/useURLParams.ts

```diff
@@ -0,0 +1,27 @@
+import { useState, useMemo } from 'react'
+
+function setURLFilterParams(filterString: string, moveNow?: boolean): string | void | boolean {
+  if (!window || !window.history) return false
+  const defaultLocation = window.location.href.split('?')[0]
+  if (!filterString) return null
+  return moveNow
+    ? (window.location.href = `${defaultLocation}${filterString}`)
+    : window.history.replaceState(null, null, `?${filterString}`)
```

Comment:
Do we need to do this manually, I thought there was already react router abstractions for this

Reply:
`window.location.href` won't always move, though will in our case
`window.history.replaceState` won't do a thing aside from changing address bar
Still better use react-router so that we don't reload the page